README.md

```diff
@@ -210,6 +210,39 @@ Command: <code>python3 -m pip install --user --upgrade pip</code>
 	Link to Solution: <a href="https://github.com/Udayraj123/OMRChecker/issues/70#issuecomment-1268094136">#70</a>
 </details>
 
+### Running in a Container Runtime (Docker or Podman)
+
+The repo contains a containerfile that can be used to run ORM checker without 
```

Comment:
typo - ORM checker

Reply:
Also, let's add a small link to Docker setup for new people who wish to try out this setup method.